Doc/tools/check-epub.py

```diff
@@ -0,0 +1,23 @@
+import sys
+from pathlib import Path
+
+
+def main() -> int:
+    if not Path("Doc").exists() or not Path("Doc").is_dir():
+        raise RuntimeError("Must run this script from the repo root")
+
+    with Path("Doc/epubcheck.txt").open(encoding="UTF-8") as f:
+        warnings = [warning.split(" - ") for warning in f.read().splitlines()]
+
+    fatal_errors = [warning for warning in warnings if warning[0] == "FATAL"]
```

Comment:
Should we just call them `errors`?

Reply:
I propose `messages`, it's more generic and [inline with the tool's docs](https://www.w3.org/publishing/epubcheck/docs/messages/)